build.sbt

```diff
@@ -62,19 +62,19 @@ lazy val microservice =
     )
     .settings(majorVersion := 0)
     .settings(PlayKeys.playDefaultPort := 9653)
+    .settings(scalafmtOnCompile := true)
```

Comment:
Does this slow down compilation (maybe touch/reformat files which then forces to recompile)?

I wanted to research Scalafmt options a little further - I think there's an option to only run it on CI/detect unformatted code, so you don't have to reformat everything constantly yourself. Also, I'm surprised there aren't any formatting changes associated with adding this here.

Reply:
There are some formatting changes in the build.sbt file. I have not noticed any slow down of compilation and we are using this in most of our other projects

Reply:
I benchmarked it. Indeed, incremental compile is efficient (when checking out all code it takes ~4 seconds to format, but then only 150ms to format one changed file).

This option however is already present in the build.sbt file, so this is a duplicate.

Reply:
Right, back to the opticians...